pinot-controller/src/main/java/org/apache/pinot/controller/api/resources/PinotRunningQueryResource.java

```diff
@@ -153,7 +154,8 @@ public Map<String, Map<String, String>> getRunningQueries(
       @ApiParam(value = "Timeout for brokers to return running queries") @QueryParam("timeoutMs") @DefaultValue("3000")
           int timeoutMs, @Context HttpHeaders httpHeaders) {
     try {
-      Map<String, List<InstanceInfo>> tableBrokers = _pinotHelixResourceManager.getTableToLiveBrokersMapping();
+      Map<String, List<InstanceInfo>> tableBrokers = _pinotHelixResourceManager
+              .getTableToLiveBrokersMapping(Optional.empty());
```

Comment:
no need to change here. add a new overload in PinotHelixResourceManager since it is a public API